### PR TITLE
manual for tcurl

### PR DESCRIPTION
**Pull request by ShanniLi**
r @kriskowal @Raynos 
cc @jcorbin

### CODE REVIEW

**Comment by Raynos** [comment]
Did you write the man page by Hand ? 

There is a tool to generate man page from read me.

https://github.com/Raynos/itape/blob/master/scripts/create-man.sh

**Comment by ShanniLi** [comment]
@Raynos Interesting ... I will update. :)

File: usage.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+    It is the same as:
+    ```
+    tcurl -p 127.0.0.1:21300 serviceName Meta::health -t meta.thrift
+    ```
+
+`-2 | --arg2 | --head content`
+    Specify the head (a.k.a. arg2) of the request. The content can be either a
+    raw string or a JSON string depending on the request format.
+
+`-2 | --arg3 | --body content`
```
Comment (Raynos) [inline usage.md]: -3

File: usage.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
+# tcurl(1) -- curl for tchannel
+
+## SYNOPSIS
+
+`tcurl` [--help] [-H] [-p] [-t]
```
Comment (kriskowal) [inline usage.md]:
```
tcurl [ -H hosts.json | -p host:port ] service Service::function [ -t thrift/ service ] ...
tcurl [ -H hosts.json | -p host:port ] service endpoint --http METHOD ...
tcurl [ -H hosts.json | -p host:port ] service endpoint --raw ...
tcurl [ -H hosts.json | -p host:port ] service endpoint ...
    ... [ --body body ] [ --head head ] [ --sk shard-key ]
tcurl [ -H hosts.json | -p host:port ] service --health ...
   ... [ --timeout ms ]
```

Comment (ShanniLi) [inline usage.md]: This seems more like the full help other than synopsis. I guess the idea is to have a very short synopsis and then a full help to follow.

**Comment by Raynos** [comment]
Let's get it in and improve later :D

**Comment by kriskowal** [comment]
:ok:

**Pull request merged**